base_suspend_security/models/res_users.py

```diff
@@ -12,8 +12,8 @@ class ResUsers(models.Model):
     def _browse(cls, ids, env, prefetch=None, add_prefetch=True):
         """be sure we browse ints, ids laread is normalized"""
         return super(ResUsers, cls)._browse(
-            [
+            models._normalize_ids([
```

Comment:
why do you need this? That's the first thing Odoo does when you browse: https://github.com/OCA/OCB/blob/12.0/odoo/models.py#L4736

Reply:
ah, but not for `_browse`